songrde/src/main/resources/templates/albums.html

```diff
@@ -33,6 +33,21 @@ <h1>Albums</h1>
         </tr>
         </tbody>
     </table>
+    <div class ="album-form">
+        <form action="/albums" method="POST">
+            <label for="title">Title:</label>
+            <input type="text" id="title" name="title" required>
+            <label for="artist">Artist:</label>
+            <input type="text" id="artist" name="artist" required>
+            <label for="songCount">Song Count:</label>
+            <input type="number" id="songCount" name="songCount" min="1" step="1" required>
+            <label for="length">Length (in seconds):</label>
+            <input type="number" id="length" name="length" min="1" step="1" required>
```

Comment:
Make sure the id of your input field is *exactly* the same as the name of the variable you're passing in to the addAlbum method in your Post request